manim/animation/composition.py

```diff
@@ -48,20 +48,18 @@ def get_run_time(self):
         return super().get_run_time()
 
     def begin(self):
-        for anim in self.animations:
-            anim.begin()
-
-    def finish(self):
-        for anim in self.animations:
-            anim.finish()
```

Comment:
I can see why this finish has to go. But if we just delete it, then the parent class' `finish` method will be called. For extra clarity, we may just want to override `finish` with a `pass` so that the parent class is not called.

Reply:
I haven't really investigated what `finish` actually does for animations, but it sounds like a good approach.